### PR TITLE
Impact lbl

### PR DESCRIPTION
**Pull request by biaobin**
For dipole nonlinear K. Brown map, coordinate transformation should use (x,xp=gambetx/gambetz), where xp is the slope.
[Benchmark_with_Elegant.pdf](https://github.com/impact-lbl/IMPACT-Z/files/8518365/Benchmark_with_Elegant.pdf)

### CODE REVIEW

**Review by qianglbl — CHANGES_REQUESTED** [review]
Please remove the print* statement.

**Review by qianglbl — COMMENTED** [review]
I just updated the Dipole.f90 following your email before see this commit.

File: src/Appl/BeamBunch.f90 [hunk 1]
```diff
@@ -400,6 +405,7 @@ subroutine drift1_BeamBunch(this,beamln,z,tau,bitype,nseg,nst,ihlf)
                                                 this%Pts1(6,i)
             enddo
 
+            print*,"biaobin is here."
```
Comment (ax3l) [inline src/Appl/BeamBunch.f90:408]: Please see Ji's comment:
```suggestion
```

Comment (qianglbl) [reply src/Appl/BeamBunch.f90:408]: The above fix is added in the new version now.

Comment (qianglbl) [reply src/Appl/BeamBunch.f90:408]: The above fix is added in the new version now.

**Pull request closed without merging**